tests/snapshots/tests.contrib.aiohttp.test_aiohttp_client.test_trace_query_string.json

```diff
@@ -15,6 +15,7 @@
       "http.status_code": "404",
       "http.status_msg": "NOT FOUND",
       "http.url": "http://localhost:8001/status/200/?k1=v1&k2=v2&k3=v3&k4=v4",
+      "language": "python",
```

Comment:
aside: I wonder if we should add this validation to the test agent itself.

We an have the test agent validate that the first span in every chunk has a language tag which matches the `X-Datadog-Header` tag.

(we still need to update these snapshots, but we can also add the validation elsewhere)